pkg/amqp/config.go

```diff
@@ -2,12 +2,11 @@ package amqp
 
 import (
 	"crypto/tls"
+	"github.com/cenkalti/backoff/v3"
```

Comment:
This should go back to its original place.

Reply:
these should be good